tests/common/plugins/memory_utilization/memory_utilization_dependence.json

```diff
@@ -296,5 +297,24 @@
       },
       "memory_check": "parse_frr_memory_output"
     }
+  ],
+  "Arista-7060CX": [
+    {
+      "name": "monit",
+      "cmd": "sudo monit validate",
+      "memory_params": {
+        "memory_usage": {
+          "memory_increase_threshold": {
+            "type": "percentage_points",
+            "value": 10
+          },
+          "memory_high_threshold": {
+            "type": "percentage_points",
+            "value": 75
```

Comment:
Could you please share the failure cases, let me check the status in our nightly runs, thanks

Reply:
thanks for your information @rgarofano-arista , could you please share more details.
Specifically, please list the failure cases from the latest run and provide the logs, I need to check it first, as the high memory threshold is related to the high memory alarm in production, we need to clarify this before making any updates, thanks

Reply:
thanks, @rgarofano-arista , could you please send the full logs to me first, you can sent it in mail, and let's sync it later, thanks